activity_heartbeating/starter.rb

```diff
@@ -1,10 +1,15 @@
 # frozen_string_literal: true
 
 require 'temporalio/client'
+require 'temporalio/env_config'
 require_relative 'my_workflow'
 
+# Load config and apply defaults
+positional_args, keyword_args = Temporalio::EnvConfig::ClientConfig.load_client_connect_options
```

Comment:
```suggestion
args, kwargs = Temporalio::EnvConfig::ClientConfig.load_client_connect_options
```

This shortening of the var names would be fairly normal in Ruby since these are the normal terms for the two forms of args